### PR TITLE
Add a subxt config lib.

### PR DESCRIPTION
**Pull request by luka-ethernal**

### CODE REVIEW

File: avail-subxt/examples/submit_data.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+use sp_keyring::AccountKeyring;
+use subxt::{tx::PairSigner, OnlineClient};
+
+use avail_subxt::*;
+
+use avail_subxt::avail::runtime_types::frame_support::storage::bounded_vec::BoundedVec;
+
+/// This example gets all the headers from testnet. It requests them in concurrently in batches of BATCH_NUM.
```
Comment (prabal-banerjee) [inline avail-subxt/examples/submit_data.rs:8]: The comment might be stale. Can we remove it and add some comment about what this script does?

File: avail-subxt/examples/max_block_submit.rs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+use avail_subxt::avail::runtime_types::frame_support::storage::bounded_vec::BoundedVec;
+use avail_subxt::*;
+use sp_keyring::AccountKeyring;
+use std::time::Instant;
+use subxt::ext::sp_core::H256;
+use subxt::{tx::PairSigner, OnlineClient};
+
+/// This example gets all the headers from testnet. It requests them in concurrently in batches of BATCH_NUM.
```
Comment (prabal-banerjee) [inline avail-subxt/examples/max_block_submit.rs:8]: Same as the other comment.

**Review by sh3ll3x3c — APPROVED** [review]

File: avail-subxt/examples/max_block_submit.rs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            .tx()
+            .sign_and_submit(&data_transfer, &signer, extrinsic_params.clone())
+            .await
+            .unwrap();
+        println!("hash #{i}: {h}");
+        hashes.push(h);
+    }
+    let end = start.elapsed();
+
+    println!("Done in {end:?}!");
```
Comment (prabal-banerjee) [inline avail-subxt/examples/max_block_submit.rs:37]: Can we verify that block was filled up? May not be in this script but in a separate script to extract block size?

**Review by aterentic-ethernal — APPROVED** [review]

**Review by prabal-banerjee — APPROVED** [review]

File: avail-subxt/examples/max_block_submit.rs (hunk 3 above)
Comment (luka-ethernal) [reply avail-subxt/examples/max_block_submit.rs:37]: Yes, it's not exactly for this script, but I've started doing this in the tests, as it requires a little more effort.

File: avail-subxt/examples/max_block_submit.rs (hunk 2 above)
Comment (luka-ethernal) [reply avail-subxt/examples/max_block_submit.rs:8]: done

File: avail-subxt/examples/submit_data.rs (hunk 1 above)
Comment (luka-ethernal) [reply avail-subxt/examples/submit_data.rs:8]: done

**Pull request merged**